docs/getting-started/github-apps/migrating-tools-to-github-apps.rst

```diff
@@ -0,0 +1,175 @@
+Migrating Your Existing Tools to Use GitHub Apps
```

Comment:
I think we should consider using a term other than "Existing tools"... After the 1.12 release, both non-GitHub app tools and GitHub app tools will exist. :) And people will probably be migrating for a while...

The problem is I don't have a great, compact alternative. Container-based tools? Image-based tools?

I could be overthinking this, and "existing" can stay, unless it triggers similar responses from anybody else.

Reply:
Second (it kinda stands to reason the opposite of tools that use github apps are tools that don't use github apps, but maybe I've been looking at this too long)